syntax = "proto3";
package proto;

enum StorageKind
{
    Storage_Undefined = 0;
    Storage_Normal = 1;
    Storage_Static = 2;
    Storage_Extern = 3;
    Storage_Local = 4;
};

message Symbol {
    bytes uuid = 1;
    uint64 ea = 2;
    string name = 3;
    StorageKind storage_kind = 4;
    bytes data_referent_uuid = 5;
    bytes code_referent_uuid = 6;
}
